### DISCUSSION TITLE
手机购买专业版后，无法授权电脑登录

### DISCUSSION THREAD

**Discussion by Mr6-wang** [getlantern/lantern-forum-cn#827] [Q&A]
手机购买专业版后可以正常使用，授权电脑登录时，输入动态码后显示已为专业版，但油管谷歌都上不去，刷新蓝灯页面又变成免费版了，点击账户管理显示没有绑定成功，请解决~

**Comment by MaxMKMax** [comment]
@Mr6-wang 您好. 请您提供我支付单据和账号邮箱. 我帮您提交问题上去解决问题. 谢谢!
